code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -358,8 +358,8 @@
 	reagents.add_reagent(/datum/reagent/drink/grenadine, 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/cola
-	name = "\improper Space Cola"
-	desc = "Cola. in space."
+	name = "\improper Coca-Cola"
+	desc = "Cola, made fresh and delivered just to you!"
```

Comment:
Freshly made cola?

Reply:
Yes.

Reply:
No.

Reply:
desc = "Some coca-cola, made best-quality and delivered just to you!"